Replace postinstall script with cross-platform cpy-cli

ahmedhesein1 opened the pull request:
#### Description of Change

This PR updates the `postinstall` script in `package.json` to use `cpy-cli` instead of the Unix-specific `cp` command. The original script (`cp node_modules/prismjs/themes/*.css src/static/css/prismjs`) failed on Windows because `cp` isn’t available, causing `yarn install` to exit with code 1. By switching to `cpy-cli`, a cross-platform Node.js tool, and adding it as a dev dependency, this change ensures the script works on Windows, macOS, and Linux.

- Changed `postinstall` to: `cpy 'node_modules/prismjs/themes/*.css' 'src/static/css/prismjs/'`.
- Added `cpy-cli` with `yarn add -D cpy-cli`.
- Tested on Windows 11 with Yarn 1.22.22; confirmed `.css` files copy to `src/static/css/prismjs/` and `yarn start` runs successfully.

This improves the setup experience for contributors across platforms, aligning with Electron’s goal of reliable tooling.

#### Checklist

- [x] This PR was not created with AI. (I wrote the code and tested it myself)

ahmedhesein1 commented:
Hi @dsanders11, I’ve opened this PR for the Releases Working Group Calendar project . Could you please review it when you have time?

Also, I’d love to collaborate further—should I join the #wg-releases Slack channel for discussions? If so, how can I get an invite?

Thanks for your guidance!

erickzhao reviewed (changes requested):
Why did you delete the entire lockfile?

ahmedhesein1 commented:
> Why did you delete the entire lockfile?

> Hi @erickzhao ,

I’ve restored the `yarn.lock` file and updated the PR. I reverted the deletion and re-ran `yarn install` to ensure it includes the new `cpy-cli` dependency while maintaining consistency for other dependencies. The setup still works as expected on Windows 11 with Yarn 1.22.22.

Thank you for catching this—I’ve learned to be more careful with lock files in the future! Please let me know if there’s anything else to address.

Best,
Ahmed (@ahmedhesein1)

erickzhao commented on package.json:
```diff
@@ -42,7 +43,7 @@
     "lint": "prettier --check \"src/**/*.js\" \"src/**/*.css\"",
     "prettier:write": "prettier --write \"src/**/*.js\" \"src/**/*.css\"",
-    "postinstall": "cp node_modules/prismjs/themes/*.css src/static/css/prismjs",
+    "postinstall": "cpy 'node_modules/prismjs/themes/*.css' 'src/static/css/prismjs/'",
```
```suggestion
    "postinstall": "node -e 'fs.cpSync(\"./node_modules/prismjs/themes/\", \"./src/static/css/prismjs/\", { recursive: true })'",
```

We might be able to get away with something like this without installing any additional deps?

erickzhao reviewed (changes requested):
The lockfile changes are still here. Did you run `yarn` after removing the line from the `package.json` file?

ahmedhesein1 commented:
> The lockfile changes are still here. Did you run `yarn` after removing the line from the `package.json` file?

> Yes, I did that and the setup still working.

dsanders11 commented:
@ahmedhesein1, this PR still needs the changes in `yarn.lock` removed, the title updated to reflect the latest state, and all commits must be verified.
